backend/app/settings.py

```diff
@@ -80,6 +80,16 @@ class Settings(BaseSettings):
     DESTINATION_INDEX: str = "destinations"
     ACTION_INDEX: str = "actions"
     USER_INDEX: str = "user"
+    CELERY_INDEX: str = "celery"
+
+    SWIPLE_CELERY_CONFIG: dict = {
```

Comment:
Wouldn't it be more convenient to split each setting in its own variable? Typically, you may not want to explicitly override the limits and multiplier.

Reply:
I used a dict from a convenience standpoint when applying the config values to the celery app here
https://github.com/Swiple/swiple/blob/e1de10e816fc024589fc6bfa4d3dc4396d102abe/backend/app/worker/app.py#L33

It's important for us to disable prefetching. The reason being that if workers are deployed on ECS or EKS and we have multiple containers/pods, we don't want tasks/jobs being pulled by a container when all of the workers are busy. Prefetch prevents this, ensure that tasks are only pull off the queue when a worker is ready to process it. 

The limits are to protect against tasks that don't finish. I think we want some timeout set. The community can always override these values in the env file like all the other settings.

What do you think?

Reply:
Yeah, my point was more about convenience when we want to configure an environment. If we want to override one of those setting (e.g. `broker_url`), we need to remember to have the whole dictionary with all other settings for which we probably want to keep the defaults (e.g. `task_default_queue` or `worker_prefetch_multiplier`).

But I understand it's a bit cumbersome in the code to have a big list of setting for each one.

Reply:
Maybe we set `task_default_queue` and `worker_prefetch_multiplier` separately outside  `settings.SWIPLE_CELERY_CONFIG`?